spec/CloudCode.spec.js

```diff
@@ -3167,4 +3167,26 @@ describe('afterLogin hook', () => {
     const query = new Parse.Query(TestObject);
     await query.find({ context: { a: 'a' } });
   });
+  it('can send email via Parse.Cloud', async done => {
```

Comment:
Can you wrap the new tests into their own `describe`? The test file is already a long sausage of test cases that should be grouped at some point.